analyzer/testdata/etc/manifest.trivy

```diff
@@ -0,0 +1,4 @@
+[

```

Comment:
I would suggest to make this file csv or yaml format, so you can append more easily than with json, wdyt?

Reply:
Aye, that is a sane thought, not sure why I went with JSON to be honest, hehe.
Will update that in a bit (after kids are in bed ;) )

Reply:
Will have to look a bit deeper into this, it seems like there is no native yaml support, and doesn't feel right to add a new dependency for this tiny feature... I'll figure something out, and I still agree with you that JSON is not the perfect format :)